src/main/java/com/iota/iri/service/TipsSolidifier.java

```diff
@@ -45,7 +54,7 @@ public void init() {
                     log.error("Error during solidity scan : {}", e);
                 }
                 try {
-                    Thread.sleep(RESCAN_TX_TO_REQUEST_INTERVAL);
+                    Thread.sleep(getRescanInterval());
```

Comment:
Not neccessary

Reply:
Also made for the possibility of configuration later

Reply:
No need to configure this. What is needed is to remove this ugly sleep and use a ScheduledExecutor.

Revert please.